### PR TITLE
NAS-101922 / 11.3 / Migrate iocage mountpoints

### PR DESCRIPTION
**Pull request by sonicaj**
This commit adds a check in check interface of iocage which would perform migration of iocage dataset mountpoint if it is not mounted under the activated pool. This would default to false for FreeBSD users and for FN/TN users, we would enforce this.

### CODE REVIEW

**Review by william-gr — APPROVED** [review]
Looks reasonable, but wait for brandon's approval

**Review by skarekrow — APPROVED** [review]

**Pull request merged**
